File: EDD 2024-2/Mod3/Excercise4.py
L=[]

while(True):
    i=int(input())
    if(i==0):
        break
    else:
        if(i in L):
            L.remove(i)
        elif(i+1 in L):
            L.remove(i+1)
        elif(i-1 in L):
            L.remove(i-1)
        else:
            L.append(i)

if (L==[]):
    L.append(0)
for e in L:
    print(e, end=' ')

File: EDD 2024-2/Mod2/Excercise4.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(){
    int n, len;
    string input;
    cin>>n;
    getline(cin>>ws, input);
    len= input.length()-1;
    for(int i=0; i< input.length()/2; i+=3){
        cout<<input[i]<<input[len-i];
    }
    if(((len)/3 +1)%2==1){cout<<input[len/2];}
    return 0;
}